services/pkg/docker-bake.hcl

```diff
@@ -35,6 +35,11 @@ target "infra-lego" {
   context = "services/pkg/lego/"
 }
 
+target "infra-lsyncd" {
+  inherits = ["_common-glibc"]
```

Comment:
I don't suppose this could be musl based?

Reply:
it could I guess